pkg/daemon/ceph/rgw/init.go

```diff
@@ -111,41 +114,15 @@ func generateConfigFiles(context *clusterd.Context, config *Config) error {
 	}
 
 	// write the mime types config
-	mimeTypesPath := getMimeTypesPath(context.ConfigDir)
-	logger.Debugf("Writing mime types to: %s", mimeTypesPath)
+	mimeTypesPath := GetMimeTypesPath(context.ConfigDir)
```

Comment:
Offtopic to this PR, but does https://github.com/rook/rook/blob/02accb5d62142de07c20842b37593c5d982c9a3d/pkg/daemon/ceph/rgw/mime.go need an update (from time to time)?